docs/webhooks/callbacks/_all.yml

```diff
@@ -278,13 +278,13 @@ properties:
       allOf:
       - $ref: https://raw.githubusercontent.com/bigcommerce/docs/main/docs/webhooks/callbacks/store_metafield_updated.yml  
   store/modifier/updated:
-    description: Fires when product modifier are edited. The `context` fields are present for only updates to overrides, not the global store. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.
+    description: Fires when a product modifier is edited for a channel locale. The `context` fields are present when you update the overrides. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.
     type: object
     properties:
       allOf:
       - $ref: https://raw.githubusercontent.com/bigcommerce/docs/main/docs/webhooks/callbacks/store_modifier_updated.yml
   store/option/updated:
-    description: Fires when product options are edited. The `context` fields are present for only updates to overrides, not the global store. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.
+    description: Fires when product options are edited for a channel locale. The `context` fields are present when you update the overrides. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.
```

Comment:
let's replace ` for a channel locale` to `context of a channel and locale`. 
Or in case you have any better suggestions, let's use those.